src/app/portlets/dot-starter/dot-starter.component.ts

```diff
@@ -10,13 +12,23 @@ import { DotToolGroupService } from '@services/dot-tool-group/dot-tool-group.ser
 })
 export class DotStarterComponent implements OnInit {
     username: string;
+    showCreateContentLink: boolean;
+    showCreateDataModelLink: boolean;
+    showCreatePageLink: boolean;
+    showCreateTemplateLink: boolean;
 
     constructor(private route: ActivatedRoute, private dotToolGroupService: DotToolGroupService) {}
 
     ngOnInit() {
-        this.route.data.pipe(pluck('username'), take(1)).subscribe((username: string) => {
-            this.username = username;
-        });
+        this.route.data
+            .pipe(pluck('userData'), take(1))
+            .subscribe(([currentUser, userPermissions]: [DotCurrentUser, DotPermissionsType]) => {
+                this.username = currentUser.givenName;
+                this.showCreateContentLink = userPermissions[PermissionsType.CONTENTLETS].canWrite;
+                this.showCreateDataModelLink = userPermissions[PermissionsType.STRUCTURES].canWrite;
+                this.showCreatePageLink = userPermissions[PermissionsType.HTMLPAGES].canWrite;
+                this.showCreateTemplateLink = userPermissions[PermissionsType.TEMPLATES].canWrite;
```

Comment:
Use `async` pipe to prevent change detection problems.

You can map all this into one object and use `ng-container` to get the whole thing like:

```
<ng-container *ngIf="yourObjectMapped$ | async as whatever">
   {{ whatever.XXX }}
</ng-container>
```